pkgs/top-level/all-packages.nix

```diff
@@ -16859,6 +16859,8 @@ in
 
     perf = callPackage ../os-specific/linux/kernel/perf.nix { };
 
+    gpio-utils = callPackage ../os-specific/linux/kernel/gpio-utils.nix { };
```

Comment:
Do we really need to build those for every kernel version? I would assume that the kernel interfaces are sort of stable across releases.

Reply:
It doesn't look like the source has changed much recently. I've moved it to just one version based on the default kernel.